### ISSUE TITLE
Multiline comment for region causes error

### ISSUE THREAD

**Issue by goldstar611** [minio/minio#15736]
## Expected Behavior
Be able to save multi line region comment without error

## Current Behavior
When using a multiline comment the following error banner is displayed:

```
Unknown sub-system This is line 2"
```

## Possible Solution
Improve the subsystem/key/value parser. I found the error message defined at https://github.com/minio/minio/blob/18dffb26e7aba09074704d169517285ac3870e86/internal/config/config.go#L682
but I don't know any golang.

## Steps to Reproduce (for bugs)

1. Set up a single node single drive test instance
2. Log into the console
3. Set the region comment to something with multiple lines
4. Click Save

## Context
<img width="802" alt="image" src="https://user-images.githubusercontent.com/13027044/191789323-0fc8b6d9-6be4-4ac9-b35f-d74ee44c6840.png">

## Regression
Don't know.

## Your Environment
* Version used (`minio --version`):
```
minio version RELEASE.2022-09-17T00-09-45Z (commit-id=d89f6af6c4ab280140627974252077fba47bbb79)
Runtime: go1.18.6 linux/amd64
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Copyright: 2015-2022 MinIO, Inc
```
* Server setup and configuration:
```
Single Node 
Single Disk
Testing instance
```
* Operating System and version (`uname -a`):
``` 
Debian 11 64-bit Intel
```

**Comment by dormanze** [comment]
In the MINIO server, since it needs to support the ability to submit multiple lines of configuration at the same time, the configuration is read by line, which requires that no \n exist in the same configuration.
![image](https://user-images.githubusercontent.com/37854724/192078081-40f14b4f-27c2-4ac2-b539-f022cef94df1.png)

**Comment by harshavardhana** [comment]
This is fixed via minio/console#2342
